prow/crier/reporters/slack/reporter.go

```diff
@@ -39,25 +39,34 @@ type slackReporter struct {
 	dryRun bool
 }
 
-func jobChannel(pj *v1.ProwJob) (string, bool) {
-	if pj.Spec.ReporterConfig != nil && pj.Spec.ReporterConfig.Slack != nil && pj.Spec.ReporterConfig.Slack.Channel != "" {
-		return pj.Spec.ReporterConfig.Slack.Channel, true
+func jobConfig(pj *v1.ProwJob) *v1.SlackReporterConfig {
+	if pj.Spec.ReporterConfig != nil && pj.Spec.ReporterConfig.Slack != nil {
```

Comment:
The ` && pj.Spec.ReporterConfig.Slack != nil` part is redundant - we just return that value, which can as well be `nil`.

Reply:
Removed, thanks!